### PR TITLE
Enable databricks support on Web

### PR DESCRIPTION
**Pull request by rohit489**

### CODE REVIEW

**Comment by carlbrochu** [comment]
Do you have screenshots of the changes?

**Comment by rohit489** [comment]
> Do you have screenshots of the changes?

AutoScaleDisabled
AutoScaleEnabled
InfoTab

File: Website/Website/auth.js [hunk 1]
```diff
@@ -427,3 +431,7 @@ function functionEnabled(roles, enableLocalOneBox) {
 
     return supportedFunctionalities;
 }
+
+function isDatabricksSparkType(sparkType) {
+    return sparkType && sparkType.toLowerCase() == 'databricks' ? true : false;
```
Comment (bryanchen-d) [inline Website/Website/auth.js]: nit: use `===`

File: Website/Website/auth.js [hunk 2]
```diff
@@ -348,6 +348,10 @@ exports.initialize = function(host) {
             res.type('application/json').send(functionEnabled(req.user._json.roles, host.conf.env.enableLocalOneBox));
         });
 
+        app.get('/api/isdatabrickssparktype', function(req, res) {
+            res.type('application/json').send(isDatabricksSparkType(process.env.DATAX_SPARK_TYPE));
```
Comment (bryanchen-d) [inline Website/Website/auth.js]: better to centralize all parameters in the config module.

**Pull request merged**